cloudflare/resource_cloudflare_access_keys_configuration_test.go

```diff
@@ -0,0 +1,59 @@
+package cloudflare
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+)
+
+func TestAccessKeysConfiguration_WithKeyRotationIntervalDaysSet(t *testing.T) {
+	name := "cloudflare_access_keys_configuration.test"
```

Comment:
let's swap this to use the `rnd := generateRandomResourceName()` pattern to avoid hardcoded dependency or resource collisions. example: https://github.com/cloudflare/terraform-provider-cloudflare/blob/a117eb773c303ebbaf657000eb59d8e736809141/cloudflare/resource_cloudflare_magic_firewall_ruleset_test.go#L17-L18